src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/PagingQueryString.feature.disabled

```diff
@@ -0,0 +1,202 @@
+            | 7            | UT Austin Extended Campus                       |
+            | 255901001    | Grand Bend High School                          |
+            | 255901044    | Grand Bend Middle School                        |
+    When sending a GET request to "/ed-fi/schools?offset=6&limit=5"
+            """
+            """
+    Then it should respond with status 200
+    And "[]"
+
+  @ignore
+  Scenario: Ensure clients can't GET information when filtering by limit and offset using invalid values
+    When sending a GET request to "/ed-fi/schools?limit=-6&offset=-1" using values as
+            | Values                       |
+            | -1                           |
+            | 'zero'                       |
+            | '5; select * from users'     |
+            | '0)'                         |
+            | '1%27'                       | 
+    Then it should respond with 400
+    And response body is
+             """
+             {
+                "detail": "The limit parameter was incorrect.",
+                "type": "urn:ed-fi:api:bad-request:parameter",
+                "title": "Parameter Validation Failed",
+                "status": 400,
+                "correlationId": "b68e4051-8ed4-4055-9e59-4f9c2ef5f4ce",
+                "errors": [
+                "Limit must be omitted or set to a value between 0 and 500."
+                ]
+             }
+             """  
+
+  ##I need a few more details on this scenario
+  @ignore
+  Scenario: Ensure clients can't GET information when querying with filter and offset using limit without offset
+    Given these schools exist:
+            | schoolId     | nameOfInstitution                               | 
+            | 5            | School with max edorgId value                   | 
+            | 6            | UT Austin College of Education Under Graduate   |
+    When sending a GET request to "/ed-fi/schools?limit=-6&offset=-1" 
+            """
+            """
+    Then it should respond with 400
+    And response body is
+            """
+            {
+                "error": "The request is invalid.",
+                "modelState": {
+                      "limit": [
+                            "Limit must be provided when using offset",
+                      ],
+                      "offset": [],
+                },
+            }
+            """
+
+  
+  @ignore
+  Scenario: Ensure clients can GET information when filtering with limits and properties
+    Given these schools exist:
+            | schoolId     | nameOfInstitution                               | 
+            | 5            | School with max edorgId value                   | 
+            | 6            | UT Austin College of Education Under Graduate   |
+    When sending a GET request to "/ed-fi/schools?nameOfInstitution=UT+Austin+College+of+Education+Under+Graduate&limit=2" 
+            """
+            """
+    Then it should respond with 200
+    And response body is
```

Comment:
There will be some cases where we need to look closely at the entire response body. However, the response body below has some problems:

1. We won't know in advance what the `id` will be.
2. Although we only expect one response, it should be in an array: the GET queries always returns arrays, even if there are _no_ items or just _one_ item in the array.
3. The value from the ODS/API includes an address, but your setup in the table above does not have any addresses.

There are ways we could make this work. But for the scope of this feature file, the details are probably not important. We want to make sure that the response contains exactly one item (no more, no less), and we want to make sure that it is the _right_ item. Since there are two schools in the database, we can simply ensure that the correct `schoolId` is in the response. Suggestion: instead of `And response body is`:

```
And response array contains 1 items
And the response array contains schoolId=6
```

(Although "1 items" is not grammatically correct, this allows us to write a single C# step definition with a variable for the actual number, and the word "items" will make sense in other contexts where we expect multiple).

Reply:
Needs to confirm the behavior of this request because when making a GET request using this query (/ed-fi/schools?nameOfInstitution=UT+Austin+College+of+Education+Under+Graduate&limit=2), I am getting the the results of the limit, and it's not filtering by nameOfInstitution